utils/util-linux.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type='text/xsl' href='interface.xsl'?>
+<interface uri="https://apps.0install.net/utils/util-linux.xml" xmlns="http://zero-install.sourceforge.net/2004/injector/interface" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://zero-install.sourceforge.net/2004/injector/interface http://0install.de/schema/injector/interface/interface.xsd http://0install.de/schema/desktop-integration/capabilities http://0install.de/schema/desktop-integration/capabilities/capabilities.xsd">
+  <name>util-linux</name>
+  <summary xml:lang="en">Util-Linux: various system utilities</summary>
+  <description xml:lang="en">Util-Linux contains miscellaneous utilities: 
+•col: filter reverse line feeds from input 
+•colcrt: filter nroff output for CRT previewing 
+•colrm: remove columns from a file 
+•ddate: converts Gregorian dates to Discordian dates 
+•getopt: parse command options (enhanced) 
+•hexdump: ascii, decimal, hexadecimal, octal dump 
+•line: read one line 
+•rename: rename files 
+•rev: reverse lines of a file or files 
+•tailf: follow the growth of a log file 
+•whereis: locate the binary, source, and manual page files for a command 
+</description>
+  <icon href="https://raw.githubusercontent.com/0install/apps/master/utils/util-linux.ico" type="image/vnd.microsoft.icon"/>
+  <icon href="https://raw.githubusercontent.com/0install/apps/master/utils/util-linux.png" type="image/png"/>
+  <category>Utility</category>
+  <homepage>http://gnuwin32.sourceforge.net/packages/util-linux-ng.htm</homepage>
+  <needs-terminal/>
+  <implementation arch="Windows-i486" id="sha1new=c120c11be458d7bc0cccbe921001cc34fb4b2d1a" langs="ca cs da de es eu fi fr hu id it ja nl pl pt_BR ru sl sv tr uk vi zh_CN" license="GPL v2 (GNU General Public License)" released="2008-09-28" version="2.14.1-3">
+    <requires interface="https://apps.0install.net/devel/gettext.xml">
+      <environment insert="bin" name="PATH"/>
+    </requires>
+    <command name="run" path="bin/whereis.exe"/>
```

Comment:
I think this command should be named `whereis` instead of `run` since there does not seem to be any specific "main" command in this feed.

Reply:
command renamed